test/hotspot/jtreg/compiler/loopopts/superword/TestDependencyOffsets.java

```diff
@@ -23,7 +23,7 @@
 
 /*
  * @test id=vanilla-A
- * @bug 8298935 8308606 8310308 8312570 8310190
+ * @bug 8298935 8308606 8310308 8312570 8310190 8342387
```

Comment:
I don't think the bug number should be added her. It refers to the bugs that this regression test will trigger.